sunrise/settings/common.py

```diff
@@ -77,7 +77,7 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'sunrise_strike_circles',
-        'USER': 'jesse',
+        'USER': env('POSTGRES_DB_NAME'),
```

Comment:
Just to clarify here, the `USER` key is actually the Postgres *user*name, not the Postgres *database* name. I'm also going to add the Postgres user's password to the `.env` file. Thanks for helping out this, it really needed to be changed.

Reply:
Sounds good, will do!